src/frontend/shortlist/src/views/CategorizeView.vue

```diff
@@ -29,9 +30,42 @@ export default {
   computed: {
     acctID() {
       return cookie.getCookie("accountid");
-    }
+    },
   },
   methods: {
+    getList() {
```

Comment:
Kind of nitpicky, but maybe use the plural
```suggestion
    getLists() {
```
just to clearly differentiate that this would get all lists? Or, even something like getListsByAccount(). I think when we do the share page there will be a similar function, but only getting one list (so that could be getListById for example).

Reply:
changed